Makefile

```diff
@@ -1,8 +1,54 @@
-PLUGIN_ID=sentiment-analysis
```

Comment:
Can you sync with Nico T to use the latest version from the plugin template?

Reply:
done in https://github.com/dataiku/dss-plugin-nlp-sentiment-analysis/pull/4/commits/2e39e513360cd79a63ff34fb9ca72856d8a7e52a